libpod/networking_pasta.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// networking_pasta.go - Start pasta(1) to provide connectivity to the container
+//
+// Copyright (c) 2022 Red Hat GmbH
+// Author: Stefano Brivio <sbrivio@redhat.com>
+//
+// +build linux
```

Comment:
You do not need this tag if you append `_linux.go` to the filename

Reply:
I didn't know that, thanks for the tip! Renamed file and dropped tag.